stories/util.js

```diff
@@ -0,0 +1,246 @@
+export function getPlaceName (place, companies) {
+  // If address is provided (i.e. for carshare station, use it)
+  if (place.address) return place.address.split(',')[0]
+  if (place.networks && place.vertexType === 'VEHICLERENTAL') {
+    // For vehicle rental pick up, do not use the place name. Rather, use
+    // company name + vehicle type (e.g., SPIN E-scooter). Place name is often just
+    // a UUID that has no relevance to the actual vehicle. For bikeshare, however,
+    // there are often hubs or bikes that have relevant names to the user.
+    const company = null
+    if (company) {
+      return `${company.label}`
+    }
+  }
+  // Default to place name
+  return place.name
+}
+
+// All OTP transit modes
+export const transitModes = [
+  'TRAM',
+  'BUS',
+  'SUBWAY',
+  'FERRY',
+  'RAIL',
+  'GONDOLA'
+]
+
+/**
+ * @param  {config} config OTP-RR configuration object
+ * @return {Array}  List of all transit modes defined in config; otherwise default mode list
+ */
+
+export function getTransitModes (config) {
+  if (!config || !config.modes || !config.modes.transitModes) {
+    return transitModes
+  }
+  return config.modes.transitModes.map(tm => tm.mode)
+}
+
+export function isTransit (mode) {
+  return transitModes.includes(mode) || mode === 'TRANSIT'
+}
+
+export function itineraryToTransitive (itin, companies) {
```

Comment:
This appears to be copied verbatim from `@opentripplanner/core-utils/map`. That method should probably just be used instead since core-utils is imported anyways.